deploy/docker/docker-entrypoint.sh

```diff
@@ -153,14 +153,36 @@ do
     fi
 done
 
-## EMQ Plugin load settings
+## EMQX Plugin load settings
 # Plugins loaded by default
 
 if [[ ! -z "$EMQX_LOADED_PLUGINS" ]]; then
     echo "EMQX_LOADED_PLUGINS=$EMQX_LOADED_PLUGINS"
     # First, remove special char at header
```

Comment:
I believe these two lines of comments were intended for old format plugins.
i.e.
```
plugin_one.
plugin_two.
```
We are now changing to `{plugin_one, true}.`
so the comments should be updated to reflect this change.
e.g.

```
# Parse plugin names and place `{plugin_name, true}.` tuples in `loaded_plugins`.
```